specification.md

```diff
@@ -183,6 +183,7 @@ mixed Agents from different vendors.
 
 OpAMP supports the following functionality:
 
+* Maintaining Agent identity.
```

Comment:
I think this is already covered by the "Status reporting" below. It says "allows the Agent to report the properties of
the Agent".

Reply:
Perhaps my wording is not right, but I wanted to say that the protocol has provisions for maintaining unique identities of the Agent instances. I think it is a small but very important feature and it goes beyond reporting properties.

Reply:
Yes, I agree.